test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/008_ATF_P_TC_navi_app_PTU_fail_ForceProtectedService_ON_StartService_encrypted_true.lua

```diff
@@ -0,0 +1,273 @@
+  EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS"} } )
+  :Do(function(_,_)
+      self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+        { requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"})
+      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
+      :Do(function()
+          local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", {requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"},
+          "files/ptu_certificate_exist.json")
+          EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = SystemFilesPath.."/PolicyTableUpdate" })
+          :Do(function(_,_data1)
+              self.hmiConnection:SendResponse(_data1.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+              self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = SystemFilesPath.."/PolicyTableUpdate"})
+            end)
+          EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+        end)
+    end)
+end
+
+function Test:TestStep_Audio_NACK()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+
+  local msg = {
+    serviceType = 10,
+    frameInfo = 1,
+    frameType = 0,
+    rpcCorrelationId = self.mobileSession.correlationId,
+    encryption = true
+  }
+  self.mobileSession:Send(msg)
+
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( data.frameType == 0 and data.serviceType == 10)
+  end
+
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 10: StartServiceNACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        commonFunctions:printError("Service 10: StartServiceACK is received")
+        return false
+      elseif data.frameInfo == 3 then
+        print("Service 10: Audio NACK")
+        return true
+      else
+        commonFunctions:printError("Service 10: StartServiceACK/NACK is not received at all.")
+        return false
+      end
+    end)
+
+  commonTestCases:DelayedExp(10000)
+end
+
+function Test:TestStep_Video_NACK()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+
+  local msg = {
+    serviceType = 11,
+    frameInfo = 1,
+    frameType = 0,
+    rpcCorrelationId = self.mobileSession.correlationId,
+    encryption = true
+  }
+  self.mobileSession:Send(msg)
+
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( data.frameType == 0 and data.serviceType == 11)
+  end
+
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 11: StartServiceNACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        commonFunctions:printError("Service 11: StartServiceACK is received")
+        return false
+      elseif data.frameInfo == 3 then
+        print("Service 11: Audio NACK")
```

Comment:
@istoimenova, Test is about 'Video' service. Seems message has to be corrected. Or I propose to use something common for all services (like for NACK)